xc/device.ml

```diff
@@ -1523,45 +1523,50 @@ let get_state ~xs ~qemu_domid domid =
 	with _ -> None
 
 let cmdline_of_disp info =
+	let combine_opts : bytes list option list -> bytes list = fun opts ->
```

Comment:
Please define a type synonym for bytes list to make this signature read a bit more easily.   For example:
```ocaml
type params = bytes list
let cmdline_of_disp info =
   let combine_opts : params option list -> params = fun opts ->
   ...
```